src/index.ts

```diff
@@ -101,9 +99,13 @@ function usage(): void {
     console.error("onlyTestTsNext and localTs are (1) mutually exclusive and (2) test a single version of TS");
 }
 
-function listen(dirPath: string, tsLocal: string | undefined): void {
-    process.on("message", (message: {}) => {
+function listen(dirPath: string, tsLocal: string | undefined, onlyTestTsNext: boolean): void {
+    // Don't await this here to ensure that messages sent during installation aren't dropped.
```

Comment:
I bet THAT was tough to figure out.

Reply:
I bet it was, but I wasn’t the one who figured it out—I adapted this comment from one in `listen`’s caller.